packages/react/src/__tests__/state.test.tsx

```diff
@@ -3,20 +3,32 @@ import React from 'react'
 
 import { DataSource, Template } from '../refs'
 import { response } from '../response'
-import { increment, setStateAction, state, toggle } from '../state'
+import { increment, lookup, setStateAction, state, toggle } from '../state'
```

Comment:
funny how we have `lookup` as well as `state` in the server side models. we should find an improved naming scheme

Reply:
yeah!